packages/integration-karma/test/accessibility/LightningElement.focus/index.spec.js

```diff
@@ -4,12 +4,11 @@ import DelegatesFocusTrue from 'x/delegatesFocusTrue';
 import DelegatesFocusFalse from 'x/delegatesFocusFalse';
 import Container from 'x/container';
 
-beforeEach(() => {
-    document.body.focus();
-});
+describe('LightningElement.focus', () => {
+    beforeEach(() => {
+        document.body.focus();
+    });
 
-// TODO [#985]: Firefox does not implement delegatesFocus
-if (!process.env.NATIVE_SHADOW) {
```

Comment:
All browsers now support `ShadowRootInit.delegatesFocus`, so we should be able to remove the `if()` wrapper for this test.

#985 is now closed since Firefox shipped it in v94.